### PR TITLE
Mirage widget look update

### PR DESCRIPTION
**Pull request by Antivortex**
This PR contains the changes in the Mirage WebView widget look - margins added for WebView to not occupy the whole screen. It also contains build automation scripts.

### CODE REVIEW

File: BuildScripts/LocalBuildSettings-template.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+	"AndroidProjectPath": "../Mirage-Android-WebView-Plugin",
+	"UnityProjectPath": "../Mirage-Widget",
+	"UnityPath": "C:/Program Files/Unity/Hub/Editor/2021.3.8f1/Editor/Unity.exe",
+	"KeystorePath": "C:/repos/Fuzzball-Fighters/FF.keystore",
```
Comment (Vukiz) [inline BuildScripts/LocalBuildSettings-template.txt:5]: no reason to commit your local settings

Comment (Vukiz) [reply BuildScripts/LocalBuildSettings-template.txt:5]: Make them independent of your environment or add to ignore

File: BuildScripts/build_plugin_and_apk.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+python3 build_and_copy_aar.py
```
Comment (Vukiz) [inline BuildScripts/build_plugin_and_apk.sh:1]: do we need both sh and bat files?

File: BuildScripts/build_plugin_and_apk.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+python3 build_unity_android.py
```
Comment (Vukiz) [inline BuildScripts/build_plugin_and_apk.sh:2]: Specify python requirements in readme or comments

File: Mirage-Widget/Assets/MirageWidget/Examples/MirageWidgetExample.cs [hunk 4]
```diff
@@ -18,5 +18,13 @@ private void Start()
 			_messageBus.AddListener(_mirageWebViewManager);
 			_mirageWebViewManager.ShowWebView("https://google.com");
 		}
+		
+		private void OnApplicationPause(bool pauseStatus)
+		{
+			if (pauseStatus)
+			{
+				Application.targetFrameRate = 30;
```
Comment (Vukiz) [inline Mirage-Widget/Assets/MirageWidget/Examples/MirageWidgetExample.cs]: Our users will not like that we are playing with their framerate settings. Why do we need that? If this is required we should set the value back. This is a really strong implicit effect

**Review by Vukiz — CHANGES_REQUESTED** [review]

File: BuildScripts/LocalBuildSettings-template.txt (hunk 1 above)
Comment (Antivortex) [reply BuildScripts/LocalBuildSettings-template.txt:5]: @Vukiz They are already added to ignore. 
LocalBuildSettings.txt is used by a script and is ignored, LocalBuildSettings-template.txt is just an example not used by any script. The user of this repo is supposed to copy this file, rename it to LocalBuildSettings.txt and set up his own local paths. I will make a doc for this.

File: BuildScripts/build_plugin_and_apk.sh (hunk 2 above)
Comment (Antivortex) [reply BuildScripts/build_plugin_and_apk.sh:1]: sh is for MacOS, bat is for Windows

File: Mirage-Widget/Assets/MirageWidget/Examples/MirageWidgetExample.cs (hunk 4 above)
Comment (Antivortex) [reply Mirage-Widget/Assets/MirageWidget/Examples/MirageWidgetExample.cs]: You are right, this is redundant.

**Review by Vukiz — DISMISSED** [review]

**Review by Vukiz — DISMISSED** [review]

**Pull request merged**